tests/gradients/core/test_hadamard_gradient.py

```diff
@@ -14,7 +14,7 @@
 """
 Tests for the gradients.hadamard_gradient module.
 """
-
+# pylint: disable=use-implicit-booleaness-not-comparison
```

Comment:
Any quick answer to where did this pylint complaint happen? It will be interesting to take a quick look at them. If it turns out implicit booleaness has been there before then we can backlog it.

Reply:
I added this. Later versions of pylint, like the one I use locally, complain about things like `assert value == []`.  But I actually do want to make sure it's a list. So this was something I added in when setting up the tests.